src/libstd/sys/unix/ext.rs

```diff
@@ -121,20 +128,60 @@ impl OsStringExt for OsString {
     }
 }
 
-// Unix-specific extensions to `OsStr`.
+/// Unix-specific extensions to `OsStr`.
 pub trait OsStrExt {
+    /// Get the underlying byte view of the `OsStr` slice.
     fn as_byte_slice(&self) -> &[u8];
```

Comment:
I would personally have expected `.as_bytes()` here, although we can also tweak it later in a future stabilization PR.